packages/kbn-optimizer/limits.yml

```diff
@@ -171,7 +171,7 @@ pageLoadAssetSize:
   stackAlerts: 31499
   stackConnectors: 80788
   streams: 10000
-  streamsApp: 19225
+  streamsApp: 24183
```

Comment:
I'm assuming I had to raise this because of the new telemetry events that I'm adding with their schemas, because I cannot really see what else I'm doing in this PR that would cause an increase to that initial bundle.